README.md

```diff
@@ -470,16 +470,59 @@ imagekitio.rename_file(
 )
 ```
 
-**8. Delete file**
+**8. Delete File**
+
 Delete a file as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/delete-file). The method accepts the file ID of the file that has to be deleted.
 
 ```ruby
 imagekitio.delete_file(
     file_id: '598821f949c0a938d57563bd'
 )
 ```
+**9. File versions**
```

Comment:
Docs need to be updated as well for all code snippets. Please submit a PR against `dev` branch here https://github.com/imagekitio/docs-keep-a-changelog/pulls